client/src/components/modules/Qualifiers.js

```diff
@@ -99,6 +107,18 @@ class Qualifiers extends Component {
   addPlayer = (key) => this.add("player", key);
   removeReferee = (key) => this.remove("referee", key);
   removePlayer = (key, user) => this.remove("player", key, user);
+  
+  promptAndAddReferee = (key) => {
+    const user = prompt("Enter a username");
```

Comment:
in the future, we could consider reusing `AddPlayerModal` (or similar) for these too instead of using `prompt`

Reply:
yeah, I was thinking about doing that later in a separate PR